cloudfoundry/server/create.sh

```diff
@@ -92,7 +92,7 @@ if [ "$schedulesEnabled" ]; then
     cat << EOF >> ./scdf-manifest.yml
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_SERVICES: $SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_SERVICES,$SCHEDULES_INSTANCE_NAME
     SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED: true
-    SPRING_CLOUD_SCHEDULER_CLOUDFOUNDRY_SCHEDULER_URL: $SCHEDULES_URL
+    SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_SCHEDULERPROPERTIES_SCHEDULERURL: $SCHEDULES_URL
```

Comment:
I think we need to keep the existing property as well in order to support the other SCDF versions (since we use the same AT branch for testing across SCDF versions)?

Reply:
Great Catch!!!!!     Thanks Ilaya!

Reply:
We need to also clear the task_execution table the DB for this environment because it failed during cleanup.